QuizMaven

```diff
@@ -0,0 +1,92 @@
+QUIZ
+Note: You could verify your quiz results online https://www.tutorialspoint.com/maven/maven_online_quiz.htm but please
+be honest and answer to this questions before getting to an answer from the site provided.
+
+1.	What of the following is true about POM?
+
+•	It is fundamental Unit of Work in Maven.
+•	It is an XML file.
+•	Both of the above
+•	None of the above.
+
+ANSWER: Both of the above
+
+2.	Which of the following configuration element is present in POM.xml?
+
+•	developers
+•	mailing list
```

Comment:
mailing list is not an element

Reply:
Found it here https://maven.apache.org/pom.html
The packaging type required to be pom for parent and aggregation (multi-module) projects. These types define the goals bound to a set of lifecycle stages. For example, if packaging is jar, then the package phase will execute the jar:jar goal. If the packaging is pom, the goal executed will be site:attach-descriptor. Now we may add values to the parent POM, which will be inherited by its children. 
Most elements from the parent POM are inherited by its children, including:

groupId
version
description
url
inceptionYear
organization
licenses
developers
contributors
mailingLists
scm
issueManagement
ciManagement
properties
dependencyManagement
dependencies
repositories
pluginRepositories
build